src/store/article.module.js

```diff
@@ -27,7 +27,7 @@ import {
   UPDATE_ARTICLE_IN_LIST
 } from "./mutations.type";
 
-const initialState = {
+const initialState = () => ({
```

Comment:
Let us return a normal object here and later use object destructuring

```suggestion
const initialState = {
```